core/src/main/resources/messages/messages_nl.properties

```diff
@@ -98,6 +100,9 @@ error.goal.update.time.before.original=Creatietijd van aangepast doel ({0}) ligt
 error.no.device.request.present=Er bestaat geen ''new device request'' voor de gebruiker met het ID ''{0}''
 error.device.request.expired=Het ''new device request'' voor gebruiker met ID ''{0}'' is verlopen
 error.device.request.invalid.password=Het tijdelijke wachtwoord voor het ''new device request'' is ongeldig. Is het verzoek misschien vervangen door een ander?
+error.device.unknown.operating.system=Besturingssysteem ''{0}'' wordt niet ondersteund.
+error.device.invalid.device.name=De apparaatnaam ``{0}`` is niet toegestaan. Een geldige apparaatnaam is niet langer dan {1} tekens en bevat geen ''{2}''.
```

Comment:
Wrong quotes

Reply:
There were a few more of these. Corrected them all.